site/public/js/docker_interface.js

```diff
@@ -64,6 +67,53 @@ function addFieldOnChange() {
     }
 }
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
It would be better to put `exported removedTicked` at the top of the file instead of putting inline ignores everywhere as per [this](https://submitty.org/developer/software_and_system_design/coding_style_guide/javascript#style-rules) documentation page.  Apply this throughout the rest of these changes.

Reply:
Fixed